docs/source/spec/validation.rst

```diff
@@ -556,98 +556,6 @@ Default severity
 Best practice validators
 ========================
 
-.. _DeprecatedAuthSchemes:
```

Comment:
These validators are no longer needed. You can just mark a protocol or auth trait as deprecated to get the same effect.